méli-mélo/th-empathy/empathy.html

```diff
@@ -92,3 +92,111 @@ <h4>Code</h4>
 <pre><code>&lt;div class="wb-frmvld provisional wb-steps quiz empathy panel-body"&gt;
     &lt;!-- Steps Quiz Form --&gt;
 &lt;/div&gt;</code></pre>
+
+<h3>Life Journey Home Page Layout</h3>
+<div class="mrgn-tp-lg container">
+	<div class="row wb-eqht-grd">
+		<div class="col-xs-12 col-md-8 empathy main-content small">
+			<div class="row wb-eqht-grd">
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Preparing for end of life</h6></a>
+					<p>Information on how to prepare, organize your affairs and leave instructions to your survivors.
+ </p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Notify of a death</h2></a>
+					<p>Programs and departments to notify to cancel benefits or return ineligible benefits.</p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<a href="#" class="stretched-link"><h2 class="h4">Benefits and Programs</h2></a>
+					<p>Applying for benefits to which you may be entitled following a death.</p><br>
```

Comment:
Why a `<br>` here and not everywhere. What is the use? If it is to position the font-awesome icon it will be preferable to put it into its own block element that don't convey any semantic, like a `<div>`

```suggestion
					<p>Applying for benefits to which you may be entitled following a death.</p>
```

Reply:
Comment addressed